src/EditorFeatures/Core.Wpf/Tags/DefaultImageMonikerService.cs

```diff
@@ -12,7 +12,7 @@
 using Microsoft.VisualStudio.Imaging;
 using Microsoft.VisualStudio.Imaging.Interop;
 
-namespace Microsoft.CodeAnalysis.Editor.Tags
+namespace Microsoft.CodeAnalysis.Editor.Tags.Implementation
```

Comment:
I would prefer to have this be `Microsoft.CodeAnalysis.Editor.Tags.Wpf` to be clear about where it lives, but it felt weird to do that and then share it with cocoa.dll. If we change to using `#if !COCOA` then we're free-er to change this.